Automattic/jetpack
# Command Palette: Load on jetpack-mu-wpcom

Fixes:
- https://github.com/Automattic/wp-calypso/issues/87329

## Proposed changes:

* Added the necessary code to load the Command Palette on wpcom.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Load the plugin
* On a simple site go to wp-admin
* Open your browser's network tab
* You should see a request to `widgets.wp.com/command-palette.js`
* The command palette does not necessarily need to load successfully, this will depend on if you have the artifact or not but isn't required for testing this PR.
* Re-test on Atomic.

If you want to test these changes with the actual command palette you're going to need to wait for my other PR. I'll update this comment once it's ready for review.
